source/connect/connection-options/connection-pools.txt

```diff
@@ -0,0 +1,221 @@
+.. _kotlin-sync-connection-pools:
+
+================
+Connection Pools
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: concurrent, request, kotlin
+
+Overview
```

Comment:
Note to tech reviewer: this overview section is essentially the same as the Java section: https://www.mongodb.com/docs/drivers/java/sync/current/connection/specify-connection-options/connection-pools/#overview